dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -97,6 +98,25 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
 
     if (try_init_hw_decoder &&
         nvjpegCreate(NVJPEG_BACKEND_HARDWARE, NULL, &handle_) == NVJPEG_STATUS_SUCCESS) {
+    // disable HW decoder for drivers < 455.x as the memory pool for it is not available
+    // and multi GPU performance is far from perfect due to frequent memory allocations
+#if NVML_ENABLED
+    char version[80];
```

Comment:
Can you access the `nvmlConstants::NVML_SYSTEM_DRIVER_VERSION_BUFFER_SIZE`? (I know the docs says it's 80, I don't know which one is better).

Reply:
Done